### DISCUSSION TITLE
Campus Expert Applications ➡️ Start Here

### DISCUSSION THREAD

**Discussion by elisemoe** [community/community#17799] [GitHub Education]
### When will applications to the program open?

Applications to the GitHub Campus Experts Program open twice a year (in February and August), and you’ll have the entire month to apply to the program. 

### What is the application process?
The application to the GitHub Campus Experts Program has two steps:
1. Fill the application form on the [application website](apply.githubcampus.expert/).
2. If your application is successful, you will receive an invitation to submit a video resume to introduce yourself and your community. 

Once we’ve reviewed your video resume, you will receive an invitation to the GitHub Campus Experts training. You will be grouped with other selected students from your region and a GitHub Campus Experts to go through the program training. You’ll have six weeks to complete your training. Once you finish your training, you’ll have access to all the benefits the program can offer!

### What are the program requirements?

The application to the GitHub Campus Experts Program has the next requirements:
1. Be at least 18 years of age.
2. Validate your student status through the [GitHub Student Developer Pack](https://education.github.com/discount_requests/student_application).
3. Be enrolled in a post-secondary formal education institution.
4. Have more than one year left as a student before graduating.
5. Be a GitHub user for at least six months.
6. Not be enrolled in the GitHub Campus Advisors Program.

### I haven’t received any updates on my application.
The team works really hard to review all the applications we receive for the program. We highly recommend you to check your [application status](https://apply.githubcampus.expert/status) every week to get updates on your application. 

### Why did I get rejected?
Due to the high volume of applications we process, we cannot share specific feedback on your application at this time. We highly recommend reading this [blog post](https://github.blog/2020-12-10-introducing-the-new-and-improved-campus-experts-program/) to get all the details about the application process and some rips to apply to the program. 

We highly encourage you to apply again in the future. I'd recommend checking back at the beginning of the next semester. 

### What can I do to improve my chances of getting selected?
We highly recommend reading this [blog post](https://github.blog/2020-12-10-introducing-the-new-and-improved-campus-experts-program/) to get all the details about the application process and some rips to apply to the program. 

### What happens when I graduate?
Once you graduate you'll become a program alum. Alumni are Campus Experts who have made valuable contributions in the past and may no longer be able to commit to the same level of participation or have graduated. To honor their contributions, we add them to the [Campus Experts Alumni site](https://githubcampus.expert/alumni) for the program so they can remain connected to the community.

### I got an error message!
Apologize for the inconvenience. Please take a look if someone else has the same trouble in the comment section below or create a comment with the error message that you're getting.

**Comment by Pheogrammer** [comment]
Also the important thing to note is that, GitHub Campus Expert applications are opened on February and August each year.

For more details you can visit https://education.github.com/experts

**Comment by olawalejuwonm** [comment]
When will it open this year?
